.dockerignore

```diff
@@ -1,4 +0,0 @@
-build
-build_docker_cross
-*.yaml
-*.yml
```

Comment:
We're not copying all the files anymore, but only the binary onto an `alpine` image. So I think ignoring files is not too useful anymore.